net.infotexture.dita-bootstrap.json

```diff
@@ -15,6 +15,22 @@
     "url": "https://github.com/infotexture/dita-bootstrap/archive/3.1.zip",
     "cksum": "1d7f07f6e60f2557ecbc966d1cf3fa22aea260a1635347a57e445c588a53feda"
   },
+  {
+    "name": "net.infotexture.dita-bootstrap",
+    "description": "HTML5 output with a basic Bootstrap template",
+    "keywords": ["HTML", "HTML5", "Bootstrap", "Bootstrap 4", "Responsive"],
+    "homepage": "https://github.com/infotexture/dita-bootstrap",
+    "vers": "3.2.0",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": "<=2.5.4"
```

Comment:
@jelovirt Looks like this `<=` syntax may not be fully supported. 🤔 

Now that this is merged, the [net.infotexture.dita-bootstrap/3.2.0](https://www.dita-ot.org/plugins#!net.infotexture.dita-bootstrap/3.2.0) plug-in page shows the correct dependency as 

> DITA-OT 2.5.4 or older

But if you select 2.5 in the version drop-down, it doesn't show up.

Am I doing something wrong, or is there an error in the version filtering logic?